#maila/custom_imu_driver:
#    ros__parameters:
# removed for composition node    
#(ref: https://github.com/ros2/rclcpp/issues/715  )

        topics:
            out_imu: "/hw/imu_raw"
            
        out_msgs_period: 100 # [milliseconds] time between each readings
        i2c_slave_address: 0x09 # [-] 